src/components/Loader/PeopleList.tsx

```diff
@@ -0,0 +1,53 @@
+import { Person } from '../../types';
+import { Loader } from './Loader';
+import { PeopleItem } from './PeopleItem';
+
+type PeoplePageProps = {
+  people: Person[];
+  error: boolean;
+  loadingPeople: boolean;
+};
+
+export const PeopleList: React.FC<PeoplePageProps> = ({
+  loadingPeople,
+  error,
+  people,
+}) => {
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {loadingPeople && <Loader />}
+
+        {error && (
+          <p data-cy="peopleLoadingError" className="has-text-danger">
+            Something went wrong
+          </p>
+        )}
+
+        {!loadingPeople && (
+          <table
```

Comment:
```suggestion
        {loadingPeople 
          ? <Loader /> 
          :  (
          <table
```
you can simplify that using the ternary operator